### ISSUE TITLE
Instances do not count towards locked balance

### ISSUE THREAD

**Issue by MHHukiewitz** [aleph-im/pyaleph#541]
`locked_amount` should actually display the required balance for the holder-tier instance. Correct calculation of the required balance only happens on the scheduler, so we need to call the scheduler API to see, which instances are scheduled and how much tokens they require, so we can show them on the `/balances` API.

![Screenshot from 2024-01-03 11-41-31](https://github.com/aleph-im/pyaleph/assets/70762838/f1d258e7-c9a7-4926-93f0-3bb9406f33a3)

**Comment by hoh** [comment]
I thought that pyaleph already computed the required balance for all resources of a user :thinking: 

Having pyaleph call the scheduler, when the scheduler calls pyaleph, seems like API spaghetti. 
Should all that logic not be in pyaleph itself ?

**Comment by MHHukiewitz** [comment]
Yeah, but then we'd need to have consensus on the scheduler instances of all the CCNs. An alternative to calling the API would be for the scheduler to post an AGGREGATE/POST every time the plan changes.
